ElasticBox/appliance-openstack.md

```diff
@@ -36,11 +54,25 @@ Follow the video for detailed help.
 3. Launch an instance of ElasticBox from the appliance image and flavor.
 
 ### Next Steps
+<<<<<<< HEAD
```

Comment:
Clean up this section. remove the `<<<<<<< HEAD`, `=======`, and `>>>>>>> CenturyLinkCloud/master` lines. Also remove the duplicate content.